### PR TITLE
Create langswitch.js

### PR DESCRIPTION
**Pull request by andrewstech**
ANDREW PR

### CODE REVIEW

**Comment by yashash1511** [comment]
image
the first letter should be capital of a functional component

**Comment by yashash1511** [comment]
image
this wont help ig

File: src/containers/applications/apps/assets/langswitch.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+        }
+      >
+        <option value="en">English</option>
+        <option value="fr">French</option>
+      </select>
+    </div>
+  );
+}
+
+export default langswitch;
```
Comment (yashash1511) [inline src/containers/applications/apps/assets/langswitch.js:22]: try making it LangSwitch

File: src/containers/applications/apps/settings.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import { useSelector, useDispatch } from "react-redux";
 import wallReducer from "../../../reducers/wallpaper";
 import { Icon, Image, ToolBar } from "../../../utils/general";
 import "./assets/settings.scss";
+import langswitch from "./assets/langswitch";
```
Comment (yashash1511) [inline src/containers/applications/apps/settings.js:6]: capital first letter of the word L

Comment (andrewstech) [reply src/containers/applications/apps/settings.js:6]: ok

**Comment by yashash1511** [comment]
If you are rendering your component using JSX, the name of that component has to begin with a capital letter otherwise React will throw an error as unrecognized tag. This convention is because only HTML elements and SVG tags can begin with a lowercase letter.

https://madanswer.com/40009/why-should-component-names-start-with-capital-letter

**Comment by andrewstech** [comment]
> If you are rendering your component using JSX, the name of that component has to begin with a capital letter otherwise React will throw an error as unrecognized tag. This convention is because only HTML elements and SVG tags can begin with a lowercase letter.

thanks

**Comment by yashash1511** [comment]
should be fine now

**Comment by andrewstech** [comment]
It works :)

**Pull request merged**